modules/react/layout/stories/examples/Flex/FlexCardResProps.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {Flex, Box} from '@workday/canvas-kit-react/layout';
+import {PrimaryButton, SecondaryButton} from '@workday/canvas-kit-react/button';
+import {type} from '@workday/canvas-kit-react/tokens';
+// eslint-disable-next-line workday-custom-rules/restricted-imports
+import {useBreakpointValues} from '@workday/canvas-kit-react/layout/lib/utils/res';
+
+// temporary placeholder until type components are added to canvas-kit
+const H3 = props => (
+  <h3
+    style={{
+      ...type.levels.body.large,
+      margin: 0,
+      fontWeight: type.properties.fontWeights.bold,
+    }}
+    {...props}
+  />
+);
+
+const Body = props => <p style={{...type.levels.body.small, margin: 0}} {...props} />;
+
+export const FlexCardResProps = () => {
+  const resStyles = useBreakpointValues({
```

Comment:
I think we should document how to use this. This example is great but a little more context might help!